packages/opencensus-core/src/metrics/export/types.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/** Properties of a Metric. */
+export interface IMetric {
+  /** Metric metricDescriptor */
+  metricDescriptor: (IMetricDescriptor|null);
+  /** Metric name */
+  readonly name: (string|null);
+  /** Metric timeseries */
+  readonly timeseries: (ITimeSeries[]|null);
+}
+
+/** Properties of a MetricDescriptor. */
+export interface IMetricDescriptor {
+  /** MetricDescriptor name */
+  readonly name: (string|null);
+  /** MetricDescriptor description */
+  readonly description: (string|null);
+  /** MetricDescriptor unit */
+  readonly unit: (string|null);
+  /** MetricDescriptor type */
+  readonly type: (MetricDescriptorType|null);
+  /** MetricDescriptor labelKeys */
+  readonly labelKeys: (ILabelKey[]|null);
+}
+
+export enum MetricDescriptorType {
+  UNSPECIFIED,
+  GAUGE_INT64,
+  GAUGE_DOUBLE,
+  GAUGE_DISTRIBUTION,
+  CUMULATIVE_INT64,
+  CUMULATIVE_DOUBLE,
+  CUMULATIVE_DISTRIBUTION,
+  SUMMARY
+}
+
+/** Properties of a LabelKey. */
+export interface ILabelKey {
+  /** LabelKey key */
+  readonly key: (string|null);
+  /** LabelKey description */
+  readonly description: (string|null);
+}
+
+/** Properties of a TimeSeries. */
+export interface ITimeSeries {
+  /** TimeSeries startTimestamp */
+  readonly startTimestamp: (Timestamp|null);
```

Comment:
I think this should be ```(Timestamp|null) -> (google.protobuf.Timestamp|null)```, @songy23 Any thoughts?

Reply:
@mayurkale22 as you mentioned here [issue 133](https://github.com/census-instrumentation/opencensus-node/issues/133) we cant use proto files, as an option I can create namespace google -> protobuf -> ...